addons/stock/models/stock_move_line.py

```diff
@@ -743,36 +743,46 @@ def get_aggregated_properties(move_line=False, move=False):
             pickings = pickings.backorder_ids
 
         for move_line in self:
+            if kwargs.get('empty') and move_line.result_package_id:
```

Comment:
As it will not only return empty moves, maybe rename the `empty` kwarg for something more meaningful ?
For example, `avoid_package_line`, `except_package`, `ignore_result_package` or something like that.